wandb/sdk/wandb_config.py

```diff
@@ -22,6 +22,66 @@
 # if this is done right we might make sure this is pickle-able
 # we might be able to do this on other objects like Run?
 class Config(object):
+    """Config object
+
+    Config objects are intended to hold all of the hyperparameters associated with
+    a wandb run and are saved with the run object when wandb.init is called.
+
+    We recommend setting wandb.config once at the top of your training experiment or
+    setting the config as a parameter to init, ie. wandb.init(config=my_config_dict)
+
+    You can create a file called config-defaults.yaml, and it will automatically be
+    loaded into wandb.config. See https://docs.wandb.com/library/config#file-based-configs.
+
+    You can also load a config YAML file with your custom name and pass the filename
+    into wandb.init(config="special_config.yaml").
+    See https://docs.wandb.com/library/config#file-based-configs.
+
+    Examples:
+        Basic usage
+        ```
+        wandb.config.epochs = 4
+        wandb.init()
+        for x in range(wandb.config.epochs):
+            # train
+        ```
+
+        Using wandb.init to set config
+        ```
+        wandb.init(config={"epochs": 4, "batch_size": 32})
+        for x in range(wandb.config.epochs):
+            # train
+        ```
+
+        Nested configs
+        ```
+        wandb.config['train']['epochs] = 4
+        wandb.init()
+        for x in range(wandb.config['train']['epochs']):
+            # train
+        ```
+
+        Using TensorFlow flags
```

Comment:
In tensorflow 2.x these api's have been deprecated and moved to tf.compat.v1.  They instead tell users to use absl-py: https://www.tensorflow.org/guide/effective_tf2#api_cleanup

Reply:
ok, I'll add adsl but leave this for now